### DISCUSSION TITLE
Turkish character problem in repository name

### DISCUSSION THREAD

**Discussion by erdincdonmez** [community/community#72732] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi.
I wrote a repository name which including Turkish character after than I created a repository but when i whant to change it it gives error..
repository name was "sil"
new name (which contained Turkish char) : boş-empyt_repository (https://github.com/erdincdonmez/bo--empyt_repository)
giving error : Repository name was not changed

**Comment by Fullovka** [comment]
Hi, @erdincdonmez!  

Based on your message, it seems you are facing an issue when trying to change the name of a repository on GitHub to include Turkish characters. You should be aware that GitHub has certain restrictions on the characters allowed in repository names. This is likely causing the error.

Your original repository name "sil" does not contain Turkish characters, and changing it to "boş-empyt_repository" with Turkish characters is causing an error. GitHub restricts the use of certain characters in repository names to comply with URL standards and for security reasons. You may need to choose a different name for your repository that doesn't include Turkish characters to avoid this error.

**Comment by erdincdonmez** [comment]
You didn't understand correctly or I didn't express it correctly.i changed to "boş-empyt_repository" ant github accepted it. And than when i trying to change secont time, i get error.

this is not important to me but I wrote because of  i thing somethings wrong.

_______________ erdincdonmez.comUdemy derslerim
 

    On Friday, October 27, 2023 at 07:22:23 PM GMT+3, Vorobyov Daniil ***@***.***> wrote:  
 
 


Hi, @erdincdonmez!

Based on your message, it seems you are facing an issue when trying to change the name of a repository on GitHub to include Turkish characters. You should be aware that GitHub has certain restrictions on the characters allowed in repository names. This is likely causing the error.

Your original repository name "sil" does not contain Turkish characters, and changing it to "boş-empyt_repository" with Turkish characters is causing an error. GitHub restricts the use of certain characters in repository names to comply with URL standards and for security reasons. You may need to choose a different name for your repository that doesn't include Turkish characters to avoid this error.
